src/main.js

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env node
+if (process.versions.node.split('.')[0] < 5) {
+  throw new Error('Only Node 5+ is supported');
+}
+require('source-map-support').install();
```

Comment:
Really? just put in package.json what version you want to use.. and have the similar thing in dockerfile and you're fine.